curve25519-dalek/src/backend/vector/neon/constants.rs

```diff
@@ -0,0 +1,1500 @@
+        ),
+        u32x4x2::new(
+            u32x4::const_new(66655380, 6430175, 471782, 11947673),
+            u32x4::const_new(30596400, 18898659, 15930721, 4211851),
+        ),
+    ])),
+    CachedPoint(FieldElement2625x4([
+        u32x4x2::new(
+            u32x4::const_new(6757410, 65455566, 13584784, 11362173),
+            u32x4::const_new(10797127, 24451471, 19541370, 29309435),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(40360156, 17685025, 18326181, 3846903),
+            u32x4::const_new(13693365, 63049479, 31900359, 23385063),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(52455038, 57513503, 22163311, 27095042),
+            u32x4::const_new(48610726, 66454160, 12085341, 26357004),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(22097042, 14063840, 6705778, 14342902),
+            u32x4::const_new(66139825, 20702105, 31279090, 7495745),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(27360710, 49314837, 18774847, 7146436),
+            u32x4::const_new(37066216, 42004961, 22409916, 10524446),
+        ),
+    ])),
+    CachedPoint(FieldElement2625x4([
+        u32x4x2::new(
+            u32x4::const_new(1497507, 33054449, 11839906, 2960428),
+            u32x4::const_new(40538463, 18884538, 25018820, 4073970),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(54484385, 43640735, 2808257, 20710708),
+            u32x4::const_new(39840730, 27222424, 21783544, 11848522),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(45765237, 48200555, 9299019, 9393151),
+            u32x4::const_new(34818188, 56098995, 13575233, 21012731),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(4265428, 49627650, 24960282, 9425650),
+            u32x4::const_new(47883651, 2797524, 11853190, 22877329),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(25008173, 64199503, 380047, 12107343),
+            u32x4::const_new(12329448, 11914399, 764281, 29687002),
+        ),
+    ])),
+    CachedPoint(FieldElement2625x4([
+        u32x4x2::new(
+            u32x4::const_new(35889734, 23047226, 4022841, 7017445),
+            u32x4::const_new(7274086, 53316179, 25100176, 15310676),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(42409427, 30270106, 6823853, 31551384),
+            u32x4::const_new(40645017, 66489807, 18021817, 32669351),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(39827134, 43680850, 28297996, 20258133),
+            u32x4::const_new(26058742, 52643238, 22238331, 21690533),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(60808002, 17499995, 30042246, 29310584),
+            u32x4::const_new(48219954, 29389518, 8680514, 17844709),
+        ),
+        u32x4x2::new(
+            u32x4::const_new(6452896, 50116553, 9532047, 26821214),
+            u32x4::const_new(44524351, 50428429, 21904953, 12608048),
+        ),
+    ])),
+]);
```

Comment:
Self: seems fine. spot-checked by perturbing constants in random places, and seeing that tests failed. One place it didn't fail: if you change the shift in `P_TIMES_16_LO` to 3 instead of 4, everything works fine. That's probably due to some approximation algorithm still succeeding